### PR TITLE
DashPay: Add Google Drive support

### PR DESCRIPTION
**Pull request by HashEngineering**
## Issue being fixed or feature implemented

NMA-665 (Spike) - Prototype code for adding Google Drive support for Profile Pictures
NMA-664 - Google Drive

## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

File: wallet/src/de/schildbach/wallet/ui/dashpay/ChooseStorageServiceDialog.kt [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+import com.bumptech.glide.request.target.Target
+import com.bumptech.glide.request.transition.Transition
+import de.schildbach.wallet.ui.ExternalUrlProfilePictureViewModel
+import de.schildbach.wallet.ui.RestoreWalletFromFileViewModel
+import de.schildbach.wallet.util.KeyboardUtil
+import de.schildbach.wallet_test.R
+import org.slf4j.LoggerFactory
+
+
+class ChooseStorageServiceDialog : DialogFragment() {
```
Comment (sambarboza) [inline wallet/src/de/schildbach/wallet/ui/dashpay/ChooseStorageServiceDialog.kt:54]: Please run optimize imports on this class.

File: wallet/src/de/schildbach/wallet/ui/dashpay/ChooseStorageServiceDialog.kt [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        private const val ARG_INITIAL_URL = "arg_initial_url"
+
+        @JvmStatic
+        fun newInstance(): ChooseStorageServiceDialog {
+            val dialog = ChooseStorageServiceDialog()
+            return dialog
+        }
+    }
+
+    private val initialUrl by lazy {
```
Comment (sambarboza) [inline wallet/src/de/schildbach/wallet/ui/dashpay/ChooseStorageServiceDialog.kt:69]: Is this being used/will be used?

Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/ui/dashpay/ChooseStorageServiceDialog.kt:69]: no, this is a copy/paste leftover along with some other stuff in this PR.

File: wallet/src/de/schildbach/wallet/ui/dashpay/ChooseStorageServiceDialog.kt (hunk 1 above)
Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/ui/dashpay/ChooseStorageServiceDialog.kt:30]: These imports are not used in this class.  removed: https://github.com/dashevo/dash-wallet/pull/546/commits/56ef5b86f79f5e887e5dba1ef2667a524a6dbe71

**Comment by HashEngineering** [comment]
This still requires some refactoring and removing comments and unnecessary code.

**Review by tomasz-ludek — APPROVED** [review]
After signing the build with the right keystore (registered on Google Play Console) it worked.
Well done!

**Pull request merged**